src/services/EdFi.DataManagementService.Api/AspNetCoreFrontend.cs

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Text.Json.Nodes;
+using EdFi.DataManagementService.Api.Core;
+using EdFi.DataManagementService.Api.Core.Middleware;
+using EdFi.DataManagementService.Api.Core.Model;
+
+namespace EdFi.DataManagementService.Api
+{
+    /// <summary>
+    /// A thin static class that converts from ASP.NET Core to the DMS facade.
+    /// </summary>
+    public static class AspNetCoreFrontend
```

Comment:
@CSR2017 The purpose of this class is a simple translation from what ASP.NET Core handlers get to a DMS core request. The idea is for DMS Core to have no ASP.NET in it. With a well defined facade to DMS Core, it will be trivial for contributors to add frontends for e.g. AWS Lambda, Azure Functions, etc.

It's a static class because it's no different from having the functions inlined in Program.cs. The dependency ICoreFacade is injected by ASP.NET core either way.

Reply:
@bradbanister 
Thanks for explaining in details.